masterlist.yaml

```
@@ -3680,6 +3680,29 @@ plugins:
         itm: 64
         udr: 11
 
+  - name: 'BostonLibrarySettlement(PRP|Robots|SS2)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/68293/'
+        name: 'Boston Public Library Settlement'
+    group: *preVisCombinedGroup
+    after:
+      - 'PRP.esp'
+  - name: 'BostonLibrarySettlement.esp'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Previsibines Repair Pack' ]
+        condition: 'active("PRP.esp") and not active("BostonLibrarySettlementPRP.esp")'
+      - <<: *patchProvided
+        subs: [ 'Automatron' ]
+        condition: 'active("DLCRobot.esm") and not active("BostonLibrarySettlementRobots.esp")'
```

Messages with the same alias are sorted alphabetically by the contents of their subs:

```suggestion
      - <<: *patchProvided
        subs: [ 'Automatron' ]
        condition: 'active("DLCRobot.esm") and not active("BostonLibrarySettlementRobots.esp")'
      - <<: *patchProvided
        subs: [ 'Previsibines Repair Pack' ]
        condition: 'active("PRP.esp") and not active("BostonLibrarySettlementPRP.esp")'
```